docs/api_example_EN.md

```diff
@@ -0,0 +1,252 @@
+# API application case
+
+Through this tutorial, you will quickly learn the API calls of PaddleSeg, and easily train, evaluate and predict semantic segmentation models. We will take BiSeNetV2 and optic disc segmentation datasets as examples to teach you step by step how to call the API to build models, datasets, loss functions, optimizers and other modules.
+
+If you are more interested in the configuration call method, you can refer to[10分钟上手PaddleSeg](https://aistudio.baidu.com/aistudio/projectdetail/1672610)tutorial。
+
+**Note**：Please fork the latest version of this project on AI studio, and then run it.
+
+**Note**：For more details of PaddleSeg API，please refer to[API文档](https://github.com/PaddlePaddle/PaddleSeg/tree/develop/docs/apis)
```

Comment:
For more details of PaddleSeg API，please refer to the [tutorial](https://github.com/PaddlePaddle/PaddleSeg/tree/develop/docs/apis).

Reply:
done